### PR TITLE
refactor(locale): fix various locale data with trailing spaces

### PR DESCRIPTION
**Pull request by matthewmayer**
Remove data which incorrectly ends with a trailing space.

in the case of ar/location/state and hr/location/street_name, this was also a duplicate of a line without a space, so removed.

### CODE REVIEW

**Comment by matthewmayer** [comment]
although this breaks the rule of keeping PRs to a single module, i think its quick to review as a batch.

**Review by ST-DDT — APPROVED** [review]
Should we add this to our normalize data script?

**Comment by matthewmayer** [comment]
> Should we add this to our normalize data script?

There are a few cases where trailing or leading spaces seem intentional such as some prefixes and suffixes. 

So perhaps not.

File: src/locales/ar/location/state.ts [hunk 1]
```diff
@@ -20,7 +20,6 @@ export default [
   'عرابة',
   'رام الله',
   'طولكرم',
-  'بئر السبع ',
```
Comment (xDivisionByZerox) [inline src/locales/ar/location/state.ts:23]: Why did you remove this entry entirly instead of stripping the reailing space?

Comment (ST-DDT) [reply src/locales/ar/location/state.ts:23]: AFAICT it is already present in line 6.

https://github.com/faker-js/faker/blob/c9284dc210b78ef289eb1967bd1e6d76be0e27aa/src/locales/ar/location/state.ts#L6

Comment (xDivisionByZerox) [reply src/locales/ar/location/state.ts:23]: Allright, I did not see this, since GH only show 3 more lines around the change area. Thanks for the clearification 👍

**Review by xDivisionByZerox — APPROVED** [review]
